app/controllers/election.js

```diff
@@ -57,6 +57,7 @@ exports.create = (req, res) =>
     title: req.body.title,
     description: req.body.description,
     seats: req.body.seats,
+    type: req.body.type,
```

Comment:
Add some checks here to make sure an election does not have `seats !== 1` when `type == normal`. (And prob some more)

Reply:
I considered it, but that check is kinda irrelevant as it makes no practical difference how many seats the normal election has. But I can put in the check to make it more even more explicit.

Reply:
Ah, okey. If it makes no difference it's fine I guess :shrug:. But yeah, it might be weird if you suddenly have a normal election with 2 seats.

Reply:
I jotted in a validator
```
    validate: {
      validator: function (v) {
        return v === 'normal' && this.seats !== 1 ? false : true;
      },
      message: 'Normal elections must have exactly one seat',
    },
```
and a corresponding test
```
  it('should return 400 when creating normal elections with more then one seat', async function () {
    passportStub.login(this.adminUser.username);
    const { body: error } = await request(app)
      .post('/api/election')
      .send({
        title: 'NormalElection',
        description: 'NormalElectionDesc',
        seats: 2,
        type: 'normal',
        useStrict: false,
      })
      .expect(400)
      .expect('Content-Type', /json/);

    error.name.should.equal('ValidationError');
    error.errors.type.message.should.equal(
      'Normal elections must have exactly one seat'
    );
    error.status.should.equal(400);
  });
  ```